website/pages/docs/drivers/docker.mdx

```diff
@@ -73,6 +73,15 @@ The `docker` driver supports the following configuration in the job spec. Only
     command = "my-command"
   }
   ```
+- `cpuset_cpus` - (Optional) CPUs in which to allow execution (0-3, 0,1).
+
+Limit the specific CPUs or cores a container can use. A comma-separated list or hyphen-separated range of CPUs a container can use, if you have more than one CPU. The first CPU is numbered 0. A valid value might be 0-3 (to use the first, second, third, and fourth CPU) or 1,3 (to use the second and fourth CPU).
```

Comment:
Can we pull this paragraph up into the same line as the one above, so that it doesn't de-dent? Also, we wrap docs at 80 cols.

Reply:
Although ideally anyone using this property should know this already, it's probably worth pointing out specifically in the docs that `cpuset_cpus` pins the workload to the CPU but doesn't give the workload exclusive access to that CPU.

Reply:
Fixed.